generators/app/templates/src/components/AuthNavBar/auth-nav-bar.component.js

```diff
@@ -1,23 +1,29 @@
 import React from "react";
-
 import { NavBar } from "@components";
 
-import { NavBarProfile } from "./children";
-
-const navigation = [
-  {
-    id: "welcome",
-    icon: "img/icon/apps.svg",
-    label: "Welcome",
-    to: "/welcome"
-  }
-];
+import { NavBarProfile, LanguageDropdown } from "./children";
 
 const AuthNavBar = props => {
+  const { t, onLanguageSelect } = props;
+  const navigation = [
+    {
+      id: "welcome",
+      icon: "img/icon/apps.svg",
+      label: t("navBar.welcome"),
+      to: "/welcome"
+    }
+  ];
   return (
     <NavBar
       navigation={navigation}
       toolbar={[
+        {
+          component: () => (
+            <LanguageDropdown {...props} onLanguageSelect={onLanguageSelect} />
+          ),
+          label: "Language",
```

Comment:
Should this label be derived from the translation service?

Reply:
Solved, I removed those labels since they're not being used.